ui/apps/platform/src/services/ComplianceScanConfigurationService.ts

```diff
@@ -272,7 +272,7 @@ export function fetchComplianceReportHistory({
     );
     return axios
         .get<ReportHistoryResponse>(
-            `/v2/compliance/scan/configurations/${id}/reports/${showMyHistory ? 'my-history' : 'history'}?${params}`
+            `/v2/compliance/scan/configuration/${id}/reports/${showMyHistory ? 'my-history' : 'history'}?${params}`
```

Comment:
It was singular. So I'm changing this.

Reply:
Ouch, singular `configuraiton` is an exception to all its plural sibling request endpoints.
Okey for this contribution, but recommend seeing if it can change before release.
Merged only last week in #12857

Reply:
➕

Reply:
Yes, it should be `configurations`. I think this might be a mistake on the backend. I'll tell Luis to edit the endpoint to be correct.